Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -2631,7 +2688,7 @@ public uint lastTriggeredInUpdate
             /// Whether the action associated with the trigger state is marked as pass-through.
             /// </summary>
             /// <seealso cref="InputActionType.PassThrough"/>
-            public bool passThrough
+            public bool isPassThrough
```

Comment:
Make an obsolete property with the old name.

Reply:
This is an internal API.

Reply:
Oh, nvm then